src/test/java/teammates/ui/webapi/FeedbackSessionLogsUpdateActionTest.java

```diff
@@ -0,0 +1,145 @@
+package teammates.ui.webapi;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackSessionLogEntryAttributes;
+import teammates.common.datatransfer.logs.FeedbackSessionLogType;
+import teammates.common.util.Const;
+import teammates.test.ThreadHelper;
+
+/**
+ * SUT: {@link FeedbackSessionLogsUpdateAction}.
+ */
+public class FeedbackSessionLogsUpdateActionTest
+        extends BaseActionTest<FeedbackSessionLogsUpdateAction> {
+
+    private static final int MIN_WINDOW_PERIOD = 2 * 1000;
+    private static final int ISOLATION_PERIOD = 5 * 1000;
+
+    @Override
+    protected String getActionUri() {
+        return Const.CronJobURIs.AUTOMATED_FEEDBACK_SESSION_LOGS_UPDATE;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Override
+    @Test
+    protected void testAccessControl() {
+        // verifyOnlyAdminCanAccess();
+    }
+
+    @Override
+    @Test
+    public void testExecute() throws Exception {
+        FeedbackSessionLogsUpdateAction action = getAction();
+
+        ______TS("creating 2 logs in the same window should return only 1 log");
+
+        // create a window exceeding max size
```

Comment:
There is no such thing as "max size" anymore. This is essentially creating multiple logs in one window.